site/docs/v1/tech/apis/authentication.adoc

```diff
@@ -6,124 +6,152 @@ description: Learn how the FusionAuth APIs authenticate requests
+
+When you create an API key, the key value is defaulted to a secure random value. However, the API key is simply a string, so you may set the key value to `super-secret-key` if you'd like. A long and random value makes a good API key in that it is unique and difficult to guess, so allowing FusionAuth to create the key value is recommended.
 
-When you create an API key the key is defaulted to a secure random value but the API key is simply a string, so you may call it `super-secret-key` if you'd like. However a long and random value makes a good API key in that it is unique and difficult to guess.
+=== Managing API Keys via the API
+
+Prior to version 1.26.0, the FusionAuth administrative user interface was the only way to create API keys. This functionality was not available through an API. Starting from version 1.26.0, API keys may be created using an API. Please refer to the link:/docs/v1/tech/apis/api-keys/[API Key API] for more information.
 
 === Create an API Key
 
-image::create-api-key.png[Create an API Key,width=1200,role=shadowed]
+image::create-api-key.png[Create an API Key,width=1200]
 
 ==== Form Fields
 
 [.api]
+[field]#Id#::
+The unique Id of this API key.
+
 [field]#Key#::
-A unique key to be utilized to authorize API requests using the `Authorization` header.
+The unique string representing the API key. This is what is presented in the `Authorization` header for requests to FusionAuth.
 
 [field]#Description#::
-An optional description of how this API key will be utilized.
+An optional description of this API key.
 
 [field]#Tenant#::
-The optional tenant to which this API key will be assigned. This value cannot be changed once the API key is created. This field is only displayed when more than one tenant exists. When you assign an API key to a tenant, you will only be able to operate on users, applications, and groups in the selected tenant.
+The optional tenant to which this API key will be assigned. This value cannot be changed once the API key is created. This field is only displayed when more than one tenant exists in the FusionAuth instance. 
++
+When you assign an API key to a tenant, any requests made with this key will only be able to operate on users, applications, groups, and other entities in the selected tenant.
 
 [field]#Endpoints#::
-Select one or more endpoints this API key will be authorized to access. Selecting no endpoints will authorize this key to all API endpoints.
+One or more endpoints this API key will be authorized to access. 
++
+Selecting no endpoints will *authorize this key for all API endpoints*.
+
+[field]#Key Manager#::
```

Comment:
Should we add any additional warning here? Maybe it is self evident already?

Reply:
Added some bold. Hard to do a callout inline. Happy to hear other suggestions.